packages/indexing/src/tables-manager.ts

```diff
@@ -292,28 +309,36 @@ export class PostgresTablesManager extends TablesManager {
    * @param tableName
    * @param args
    */
-  override async hasMidIndices(tableName: string, args: IndexModelArgs): Promise<boolean> {
+  override async assertHasMidIndices(tableName: string, args: IndexModelArgs): Promise<void> {
     const expectedIndices = defaultIndices(tableName).indices.flatMap((index) => index.name)
     if (args && args.indices) {
       for (const index of args.indices) {
-        expectedIndices.push(fieldsIndexName(index, tableName))
+        expectedIndices.push(fieldsIndexName(index, tableName).toLowerCase())
       }
     }
-    const sqlIndices = expectedIndices.map((s) => `'${s}'`)
-    const actualIndices = await this.dataSource.raw(`
+    const indicesResult = (
+      await this.dataSource.raw<PgIndexResult>(`
 select *
 from pg_indexes
 where tablename like '${tableName}'
-and indexname in (${sqlIndices});
-  `)
-    return expectedIndices.length == actualIndices.rowCount
+    `)
+    )
+    const actualIndices = indicesResult ? indicesResult.rows.map((row) => row.indexname) : []
```

Comment:
should we also `toLowerCase()` here as well?

Reply:
pg already has them lowercased.

Reply:
Yeah, we could future proof by just moving the lower casing into the check function.